.bazelrc

```diff
@@ -1,29 +1,61 @@
-startup --expand_configs_in_place
-
-common --@rules_scala_annex//rules/scala:scala-toolchain=zinc_3
-common:ci --color=yes
+# Java options
+build --java_language_version="21"
+build --java_runtime_version="remotejdk_21"
+build --javacopt="-source 21 -target 21"
+build --tool_java_language_version="21"
+build --tool_java_runtime_version="remotejdk_21"
 
+# Other options
+build --experimental_use_hermetic_linux_sandbox
+build --experimental_worker_cancellation
+build --experimental_worker_multiplex_sandboxing
```

Comment:
I think we'll want to disable this sandboxing until we figure out the multiplex sandbox bug.

Reply:
Since we fixed the bug, I re-enabled it.